lib/ansible/modules/system/mount.py

```diff
@@ -85,16 +75,14 @@
         does not allow specifying alternate fstab files with mount so do not
         use this on OpenBSD with any state that operates on the live
         filesystem.
-    required: false
     default: /etc/fstab (/etc/vfstab on Solaris)
   boot:
-    version_added: 2.2
     description:
       - Determines if the filesystem should be mounted on boot.
       - Only applies to Solaris systems.
-    required: false
-    default: yes
-    choices: ["yes", "no"]
+    type: bool
+    default: 'yes'
```

Comment:
I would user double quotes to keep the same style like for `version_added` from the top of the `DOCUMENTATION` block.